.github/workflows/auto-cherry-picker.yaml

```
@@ -0,0 +1,47 @@
+name: Auto Cherry-Picker
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      PR_number:
+        description: The PR number to cherry-pick
+        type: string
+        required: true
+
+jobs:
+  inputs:
+    if: github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    outputs:
+      merge_commit: ${{ steps.step1.outputs.merge_commit }}
+    steps:
+      - id: step1
+        name: Get merge commit
+        run: |
+          COMMIT=$(gh pr view --repo "${{ github.repository }}" "${{ github.event.pull_request.number || inputs.PR_number }}" --json mergeCommit --jq '.mergeCommit.oid')
+          echo "merge_commit=$COMMIT" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  picker:
+    if: (github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'needs-cherrypick') ) || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    needs: inputs
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ needs.inputs.outputs.merge_commit }}
+
+      - name: Cherry-Pick
+        run: |
+          git config --local user.email "pantsbuild+github-automation@gmail.com"
+          git config --local user.name "Worker Pants (Pantsbuild GitHub Automation Bot) "
```

```suggestion
          git config --local user.name "Worker Pants (Pantsbuild GitHub Automation Bot)"
```

But also... spelling out the meaning makes it less fun 😅  
```suggestion
          git config --local user.name "Worker Pants"
```

I'm assuming these commits will connect back to the bot's GitHub account, and so clicking through could explain it?

---

I'd rather it be obvious...